src/lib.rs

```diff
@@ -128,7 +125,7 @@
 //! The `format` attribute can be used to customize the formatting of cells. It is an easier alternative to creating a custom renderer when you just want to customize some basic formatting.
 //!
 //! - **`precision`** - Specifies the number of digits to display after the decimal point. Only works for numbers.
-//! - **`string`** - Specifies a format string. Currently only used for `NaiveDate`, `NaiveDateTime` and `NaiveTime`. See [`chrono::format::strftime`] for more information.
+//! - **`format_string`** - Specifies a format string. Currently only used for `NaiveDate`, `NaiveDateTime` and `NaiveTime`. See [`chrono::format::strftime`] for more information.
```

Comment:
This is fine for now, but should likley clarify that the format_string means different things in different contexts (one format string format for `chrono`, and some other format for `time` for example

Reply:
Restructured the docs a bit